### ISSUE TITLE
Баг с одеждой

### ISSUE THREAD

**Issue by YungLains** [LambdaRP/Bug-Tracker-LambdaRP#57]
При выходе с сервера из инвентаря одежды пропадают маска и шлем
Возможно пропадает и еще что-то, но это не проверял

**Comment by XXanderWP** [comment]
Стоит учитывать, что если шлем очки и прочее были сбиты с модельки (например ударом) - то в инвентаре они не появятся, у системы снятия одежды немного другой метод работы.

**Comment by YungLains** [comment]
Нет. Они просто пропадают через время бывают)15.06.2018, 01:33, "XanderWP" <notifications@github.com>:Стоит учитывать, что если шлем очки и прочее были сбиты с модельки (например ударом) - то в инвентаре они не появятся, у системы снятия одежды немного другой метод работы.

**Comment by YungLains** [comment]
У меня есть готовый мотокостюм. Шлем и подшлемник я снимаю, когда с кем-то общаюсь. Если выйти с сервера, то пропадает что-нибудь. 15.06.2018, 01:33, "XanderWP" <notifications@github.com>:Стоит учитывать, что если шлем очки и прочее были сбиты с модельки (например ударом) - то в инвентаре они не появятся, у системы снятия одежды немного другой метод работы.
